urllib3/response.py

```diff
@@ -284,9 +299,10 @@ def stream(self, amt=2**16, decode_content=None):
             If True, will attempt to decode the body based on the
             'content-encoding' header.
         """
+        self._init_decoder()
         if self.chunked:
             for line in self.read_chunked(amt):
-                yield line
+                yield self._decode(line, decode_content, True)
```

Comment:
In the future, it'd be nice to supply a named var or param for magic values like True.